src/consumer/component/link-generator.js

```diff
@@ -49,16 +48,20 @@ function _getLinkContent(filePath: string, importSpecifier?: Object): string {
         // @todo: implement
       }
     }
+    if (filePath.endsWith('.st.css')) {
```

Comment:
i think this logic should be inside get-ext.js and just return st.css in this case (instead of return css)
Then we should replcae line 33:
```js
const fileExt = path.extname(filePath).replace('.', '');
```
with 
```js 
const fileExt = getExt(filePath);
```

We will probably need to change this line:
```js
return `${DEFAULT_INDEX_NAME}${path.extname(mainFile)}`;
```
to use the getExt as well

Reply:
Agree, I will change getExt to support this exception of .st.css